tests/unit-tests/test-class-admin.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+class Sensei_Class_Admin_Test extends WP_UnitTestCase {
+
+	/**
+	 * Constructor function
+	 */
+	public function __construct() { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found
+		parent::__construct();
+	}
+
+	/**
+	 * Setup function
```

Comment:
Generally, each sentence in a documentation comment should end in a period, even if it's the last sentence on the line. That includes this line, the line below, and several others in this file.

This does not include lines like `@return void`. That is not a sentence, and so it doesn't need to end with a period.

Reply:
Perfect! All fixed on this PR =)